gpii/node_modules/flowManager/test/CaptureTests.js

```diff
@@ -72,10 +81,18 @@ fluid.defaults("gpii.tests.flowManager.capture.tests", {
             name: "Simple system capture",
             tests: [{
                 name: "Check for existing FakeMag Settings",
-                expect: 2,
+                expect: 4,
                 sequence: [{
-                    funcName: "gpii.tests.flowManager.capture.adjustSolutions",
-                    args: ["{config}.server.flowManager.solutionsRegistryDataSource"]
+                    event: "{config}.server.flowManager.solutionsRegistryDataSource.events.solutionsRegistryReady",
```

Comment:
line 86-95 is to verify the solution registry has been populated correctly, which is not a part of testing the capture tool. If these events are in place to make sure the capture test starts when SR is ready, having one of these events in the test sequence should be enough.

Reply:
Yes, it's important is to wait until the SRDS creation sequence is done before running the tests. Given that, I've kept the solutionRegistryReady as the appropriate event for these tests.